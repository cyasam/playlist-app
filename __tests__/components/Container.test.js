import React from 'react'
import { shallow } from 'enzyme'
import Container from '../../src/scripts/components/Container'

describe('Container Component', () => {
  const setup = () => {
    const wrapper = shallow(<Container />)
    return {
      wrapper
    }
  }

  it('renders properly', () => {
    const { wrapper } = setup()
    expect(wrapper).toMatchSnapshot()
  })

  it('has div called container', () => {
    const { wrapper } = setup()
    expect(wrapper.find('.container').exists()).toBe(true)
  })
})
