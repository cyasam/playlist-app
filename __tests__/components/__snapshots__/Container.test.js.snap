// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component renders properly 1`] = `
<div
  className="container"
/>
`;
